chili-recipe.txt

```diff
@@ -12,7 +13,8 @@ Ingredients:
  - One 15-ounce can pinto beans, drained and rinsed
 
 Directions:
- 1. Brown the ground beef in a large bot and drain off the excess fat.
+ 1. Brown the ground beef in a large bot with bacon and drain off the excess 
+    fat - even though doing that is a sin.
```

Comment:
To be frank, not even the excess fat should be drained. Let it give the chili a real taste.